modules/Expectation.js

```diff
@@ -334,8 +334,8 @@ class Expectation {
 
   toExclude(value, compareValues, message) {
     assert(
-      isArray(this.actual) || typeof this.actual === 'string',
-      'The "actual" argument in expect(actual).toExclude() must be an array or a string'
+      isArray(this.actual) || isObject(this.actual) || typeof this.actual === 'string',
```

Comment:
i wonder if there's a way we could make `toExclude` call into `toInclude`, or have them share a core function, rather than duplicating the checks?